bc_obps/registration/fixtures/mock/userOperator.json

```diff
@@ -7,7 +7,8 @@
       "role": "admin",
       "status": "Approved",
       "verified_at": "2024-02-26 06:24:57.293242-08",
-      "verified_by": "ba2ba62a121842e0942aab9e92ce8822"
+      "verified_by": "ba2ba62a121842e0942aab9e92ce8822",
+      "user_friendly_id": 1
```

Comment:
Needed to add this to the fixtures as the fixtures are loaded after the migration so the function that adds this to existing user operators won't run on these.